### PR TITLE
make yarn install easier for use, fixes #3

### LINKED ISSUE #3: Make dependency installation easier
At the moment a user needs to run `yarn` in three different directories to install the dependencies.
Make it easier for user

### PR DESCRIPTION
**Pull request by rishabh1403**
Made `yarn` install easier for user
- Added post-install scripts in root and server
- It installs dependencies in server & client post installation in the current directory

### CODE REVIEW

**Review by ravikp7 — COMMENTED** [review]
@rishabh1403 great! it should get installation easy.
- have you changed any dependency also?
- Should the yarn.lock update this time? Am I missing something here?
Let me know your thoughts :)

**Comment by rishabh1403** [comment]
@ravikp7 Haven't changed the dependency.

File: package.json [hunk 1]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "cd server && yarn lint && cd ../client && yarn lint"
+    "lint": "cd server && yarn lint && cd ../client && yarn lint",
+    "postinstall": "cd server && yarn"
```
Comment (nimish-gupta) [inline package.json]: @rishabh1403 why are you not installing client package over here?

File: server/package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint ./"
+    "lint": "eslint ./",
+    "postinstall": "cd ../client && yarn"
```
Comment (nimish-gupta) [inline server/package.json]: @rishabh1403 why are you installing client packages from server package (server is totally independent of client)?

File: package.json (hunk 1 above)
Comment (rishabh1403) [reply package.json]: @nimish-gupta it was not switching directories and kept on running yarn in an infinite loop

File: server/package.json (hunk 2 above)
Comment (rishabh1403) [reply server/package.json]: Can we have multiple post installs ?

File: package.json (hunk 1 above)
Comment (nimish-gupta) [reply package.json]: @rishabh1403 you can create two separate scripts like 
`lint:client` and `lint:server` and then in `lint: yarn lint:client && yarn lint:server`

File: server/package.json (hunk 2 above)
Comment (nimish-gupta) [reply server/package.json]: No you can not have, but is this necessary to install client packages from server?

Comment (rishabh1403) [reply server/package.json]: Nope, fixed it

File: package.json (hunk 1 above)
Comment (rishabh1403) [reply package.json]: @nimish-gupta got it

**Comment by nimish-gupta** [comment]
LGTM.

**Pull request merged**